Questions!!

ichen2919 opened the pull request:
This is a very rough first draft but I'm already having a little trouble. Since I know that I will be using particles for my project, I thought that I should first translate the previous particles exercise from p5js to Processing. I'm having trouble with making the array and the for loop working. I commented my questions in the sketch as well as my plan for the next steps of the project.

ichen2919 committed:
Development

ichen2919 committed:
Isabel's Final

This is a very rough first draft but I'm already having a little
trouble. Since I know that I will be using particles for my project, I
thought that I should first translate the previous particles exercise
from p5js to Processing. I'm having trouble with making the array and
the for loop working. I commented my questions in the sketch as well as
my plan for the next steps of the project.

ichen2919 committed:
Push 2

I was finally able to create a square using and array and objects
created from the class. Now I am trying to get the mouse to break the
particles apart before I use the kinect

LadyK commented:
Do you still have comments on making an array and for-loop work together? I just ask because I see them working together in your code.

ichen2919 commented:
Do you mean adding comments about how it works?
On Dec 12, 2015 5:53 PM, "LadyK" notifications@github.com wrote:

> Do you still have comments on making an array and for-loop work together?
> I just ask because I see them working together in your code.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/IDMUY1133BCreativeCodingAssignments/Final_Project/pull/12#issuecomment-164200175
> .

LadyK commented:
Questions, sorry. Do you still have questions about how to make an array and a for-loop work together?

ichen2919 commented:
It works now! But now I am struggling with adding functions for each
individual particle.
On Dec 12, 2015 6:08 PM, "LadyK" notifications@github.com wrote:

> Questions, sorry. Do you still have questions about how to make an array
> and a for-loop work together?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/IDMUY1133BCreativeCodingAssignments/Final_Project/pull/12#issuecomment-164200931
> .

LadyK commented:
The particle is built from a class, correct?

If so, create the method them within the class, and call it for each object through the for loop in the main sketch.

ichen2919 commented:
So call all the methods within the for loop?
On Dec 12, 2015 6:14 PM, "LadyK" notifications@github.com wrote:

> The particle is built from a class, correct?
> 
> If so, create the method them within the class, and call it for each
> object through the for loop in the main sketch.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/IDMUY1133BCreativeCodingAssignments/Final_Project/pull/12#issuecomment-164201158
> .

LadyK commented:
If you want to call that method of each particle, yes.

ichen2919 committed:
Push 3

The dots move away but they are not consistent with the mouse placement
and they sometimes move away in rows, not individually.

ichen2919 committed:
Final Push

Here is my final push before the presentation!
